collector/cpu.go

```diff
@@ -85,7 +122,7 @@ type Win32_PerfRawData_PerfOS_Processor struct {
 	PercentUserTime       uint64
 }
 
-/* NOTE: This is an alternative class, but it is not as widely available. Decide which to use
+// NOTE: This is an alternative class, but it is not as widely available. Decide which to use
```

Comment:
I think the "Decide which to use" part of comment should be dropped since this patch makes use of it.

Reply:
I will remove it